website/pages/downloads/style.css

```
@@ -1,4 +1,16 @@
 #p-downloads {
+  /** DEBT: short-term color override until an upgread to the most recent product downloader */
```

Super minor nit here, just to add a link the Asana project for the `product-downloads-page` rollout. I also backlinked to this PR on [the `packer`-specific task](https://app.asana.com/0/1100423001970639/1199639822462636/f).

```suggestion
  /** DEBT: short-term color override until we upgrade to the most recent product downloader.
  This will be handled as part of the "Redesigned Downloads Pages" project:
  https://app.asana.com/0/1179659922742997/1199639822462636 */
```